.github/workflows/e2e.yaml

```diff
@@ -31,6 +31,7 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       simd-tag: ${{ steps.get-tag.outputs.simd-tag }}
+      relayer: ${{ steps.get-relayer.outputs.relayer }}
```

Comment:
pretty sure it explodes due to me having passed it as `relayer-tag` instead of `relayer-type` :face_with_head_bandage:

Reply:
yeah looks like it's applying the tag `rly`(the type) to the `damiannolan/rly` image which doesn't exist (we are using latest)

Reply:
Yeah, that looks like its it alright!